testnet/validator upgrade 0.6.0 - 0.7.0.md

```diff
@@ -0,0 +1,168 @@
+# Validator Upgrade Guide for laconic_81337-5 Testnet v0.6.0 -> v0.7.0
+
+This guide assumes you have followed the instructions to crete a systemd service or docker container validator node v0.6.0 and you perform the upgrade on a machine running v0.6.0
+
+It is highly recommended to make the backup of your datadir after you stop v0.6.0 and before you start v0.7.0. Unless you changed your datadir, it should be located in `~/.laconicd`
+
+## Systemd service
+Skip this section if you use docker
+
+This is very similar to building v0.6.0. We keep v 0.6.0 running until v0.7.0 is built and only after the successful build we should replace 0.6.0 binary with v0.7.0. This is to avoid jailing your validator for long downtime.
+The general upgrade plan is the following:
+  1. Install Go v1.19 (we used 1.18 for v0.6.0)
```

Comment:
I would pin to a specific version like 1.19.4

Reply:
It is 1.19.5 later in the documentation